go/config/enclave_config.go

```diff
@@ -100,3 +104,121 @@ func DefaultEnclaveConfig() *EnclaveConfig {
 		GasLimit:                  new(big.Int).SetUint64(params.MaxGasLimit / 6),
 	}
 }
+
+func NewConfigFromFlags(cliFlags map[string]*flag.TenFlag) (*EnclaveConfig, error) {
+	flagsTestMode := false
+
+	// check if it's in test mode or not
+	val := os.Getenv("EDG_TESTMODE")
+	if val == "true" {
+		flagsTestMode = true
```

Comment:
Added the prints! When you say to log, you mean to use the logger right ? Is it ok if we don't use the logger though? Asking because in the flow of the main application, the logger is only created at the container and I think probably this is ok with just the stdou print ? wdyt ?

Reply:
yeah, print should be fine